packages/cli/src/lib/snapshot/reportViewer/reportViewerInterfaces.ts

```diff
@@ -0,0 +1,9 @@
+import {ResourceSnapshotsReportOperationModel} from '@coveord/platform-client';
```

Comment:
I'd name the file `reportViewerDataModels`.

It's a bit antinomic to have an exported type in a file named fooInterfaces :D